layers/meta-resin-raspberrypi/recipes-core/images/resin-image.bbappend

```diff
@@ -8,15 +8,25 @@ SDIMG_KERNELIMAGE_raspberrypi3-64 ?= "kernel8.img"
 # Customize resinos-img
 RESIN_IMAGE_BOOTLOADER_rpi = "bcm2835-bootfiles"
 RESIN_BOOT_PARTITION_FILES_rpi = " \
-    ${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin:/${SDIMG_KERNELIMAGE} \
     bcm2835-bootfiles:/ \
     "
 
 python overlay_dtbs_handler () {
     # Add all the dtb files programatically
     if d.getVar('SOC_FAMILY', True) == 'rpi':
         kernel_imagetype = d.getVar('KERNEL_IMAGETYPE', True)
+        sdimg_kernelimage = d.getVar('SDIMG_KERNELIMAGE', True)
```

Comment:
This needs some description in the commit. I'm not sure I understand what is meant to happen here. As well, we can assume that we use uboot - we will not switch around.

Reply:
One-way move to u-boot makes things much simpler here. So I'll update the patch for that.